test/integration/spec/util/insightspublisher.js

```diff
@@ -22,6 +22,8 @@ const tokens = new Map([
 ]);
 
 describe('InsightsPublisher', () => {
+  this.timeout(30000);
```

Comment:
@markandrus In order to use `this` here, I think you should convert `describe('InsightsPublisher', () => {...})` to `describe('InsightsPublisher', function() {...})`. I saw that Travis CI was failing here.